packages/components/tests/dummy/app/templates/components/card.hbs

```diff
@@ -77,7 +78,7 @@
 
 <section>
   <p class="dummy-paragraph">
-    <h3 class="dummy-h3">Accessibility</h3>
+    <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
```

Comment:
It was already like this, but can you fix this? (move the `<h3>` out of/before the `<p>` block.
Thanks